build/index.js

```diff
@@ -52,7 +52,7 @@ async function buildCommunes(regions, departements, arrondissements, population,
 }
 
 async function buildEPCI() {
-  const rows = await extractEPCI(getSourceFilePath('epcicom2020.xlsx'))
+  const rows = await extractEPCI(getSourceFilePath('epcicom.xlsx'))
```

Comment:
maybe indicate in source code or README where this xlsx is recuperated (because sadly the location of these data changes regularly..)

Reply:
Done in the new commit